tests/e2e/magic-setter/test.php

```diff
@@ -1,4 +1,6 @@
-<?php // lint >= 7.4
+<?php declare(strict_types = 1);
+
+// lint >= 7.4
```

Comment:
Nope, we don't want this change, php-parallel-lint doesn't understand this

Reply:
Reverted